### PR TITLE
OEL-1158: Webtools paragraphs

### PR DESCRIPTION
**Pull request by abel-santos-corral**
### Description

The objective of this task is to enable the paragraphs using webtools, add those paragraphs in the OE Page and then add some examples.

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,10 +11,12 @@
         "drupal/core-recommended": "^9.1",
         "drupal/default_content": "^2.0@alpha",
         "drupal/description_list_field": "^1.0@alpha",
+        "drupal/devel": "^4.1",
```
Comment (drishu) [inline composer.json]: if you really need this put it in require dev, I doubt that this is a profile dependency

File: modules/oe_showcase_page/oe_showcase_page.info.yml [hunk 2]
```diff
@@ -6,12 +6,17 @@ package: OpenEuropa Showcase
 dependencies:
   - drupal:ckeditor
   - drupal:editor
+  - drupal:json_field
   - drupal:menu_ui
   - drupal:pathauto
   - oe_paragraphs:oe_paragraphs_banner
+  - oe_paragraphs:oe_paragraphs_chart
```
Comment (drishu) [inline modules/oe_showcase_page/oe_showcase_page.info.yml:13]: add all these newly enabled modules also to the oe_showcase.info.yml file

File: modules/oe_showcase_default_content/content/file/0e8be6ce-cf6b-4e7a-8990-3628cda4459a.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+_meta:
```
Comment (drishu) [inline modules/oe_showcase_default_content/content/file/0e8be6ce-cf6b-4e7a-8990-3628cda4459a.yml:1]: why do you need these files ?

**Review by drishu — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (abel-santos-corral) [reply composer.json:14]: I have to eliminate it, brute of me. I left it intentionally to do a QA on the reviewer ;-)

File: modules/oe_showcase_default_content/content/file/0e8be6ce-cf6b-4e7a-8990-3628cda4459a.yml (hunk 3 above)
Comment (abel-santos-corral) [reply modules/oe_showcase_default_content/content/file/0e8be6ce-cf6b-4e7a-8990-3628cda4459a.yml:1]: We need these files for the menu link entries. We generate the files (e.g 0e8be6ce-cf6b-4e7a-8990-3628cda4459a.yml) and then we have the link in the node. So those two elements work together to render later a link in the Main Administration menu. In this case we are introducing a menu link for a new page with webtools paragraphs.

Comment (abel-santos-corral) [reply modules/oe_showcase_default_content/content/file/0e8be6ce-cf6b-4e7a-8990-3628cda4459a.yml:1]: In first I thought I needed that file, but later I tryed to deploy without it and it worked fine whithout the file, so I deleted it.

**Review by drishu — DISMISSED** [review]

**Review by escuriola — DISMISSED** [review]

**Review by Maxfire — DISMISSED** [review]

File: modules/oe_showcase_default_content/content/file/0e8be6ce-cf6b-4e7a-8990-3628cda4459a.yml (hunk 3 above)
Comment (drishu) [reply modules/oe_showcase_default_content/content/file/0e8be6ce-cf6b-4e7a-8990-3628cda4459a.yml:1]: we tested and it actually is needed for the thumbnail in media admin page

**Review by drishu — APPROVED** [review]

**Review by escuriola — APPROVED** [review]

**Pull request merged**